rsts/concepts/data_management.rst

```diff
@@ -4,9 +4,28 @@
 Understand How Flyte Handles Data
 #################################
 
+Let's understand how Flyte handles data management.
```

Comment:
```suggestion
Let's understand how Flyte handles data.
```
^ this seems just like the title. I think it's better if we don't have this line.